addons/jest/package.json

```diff
@@ -34,11 +34,14 @@
     "@storybook/theming": "5.1.0-alpha.20",
     "core-js": "^2.6.5",
     "global": "^4.3.2",
-    "prop-types": "^15.7.2",
     "react": "^16.8.4",
     "upath": "^1.1.0",
     "util-deprecate": "^1.0.2"
   },
+  "devDependencies": {
+    "@storybook/api": "^5.1.0-alpha.20",
```

Comment:
Are there devDependencies though? I think they are regular dependencies.

Reply:
I thought they need only for type info, they don`t need in regular dependencies. But they also imported in dist/**.d.ts files. Ok, I`ll move them to regular dependencies.